.gitignore

```diff
@@ -1,3 +1,4 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+test/regdump.reg
```

Comment:
Can you also remove `test/regdump.reg` from the repository (`git rm test/regdump.reg`).